src/pages/using-spark/guides/validation-messaging.mdx

```diff
@@ -0,0 +1,106 @@
+---
+  title: Validation Messaging
+---
+import validationAnatomy from '../../../images/guides/validation-anatomy.png';
+import validationAnatomy2x from '../../../images/guides/validation-anatomy@2x.png';
+export const validationAnatomySet = `${validationAnatomy} 560w, ${validationAnatomy2x} 1120w`
+
+import {
+  SprkHeading,
+} from '@sparkdesignsystem/spark-react';
+
+<span id="validation-messaging-guide" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h1" additionalClasses="sprk-u-mbl sprk-u-Measure" isPageTitle>
+  Validation Messaging Guide
+</SprkHeading>
+
+<span id="main-takeaways" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Main Takeaways
+</SprkHeading>
```

Comment:
Headings look like this instead of the normal `#` because We wanted a h3 style with h2 semantics

Reply:
love this